src/pages/ProductDetail/index.js

```diff
@@ -2,55 +2,38 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 import { getAuthRequest } from '../../api/getAuthRequest'
+import Layout from '../../components/Layout'
 import Header from '../../components/Header'
+import Loader from '../../components/Loader'
+import { H1 } from '../../components/Typography'
+import ProductDetailComponent from './components/ProductDetail'
 
 class ProductDetail extends React.Component {
   state = {
     isLoading: true, // stop confusing users, what happening until fetch data
-    productData: {},
+    productData: [],
   }
 
   async componentDidMount() {
     const { match } = this.props
 
-    const productData = await getAuthRequest(`skus/${match.params.productId}`)
-    console.log(productData)
+    const productData = await getAuthRequest(
+      `skus/${match.params.productId}?include=prices`
+    )
+
     this.setState({ productData, isLoading: false })
```

Comment:
Default state.productData is [] while typeof productData (const defined in this fn) is object. More suitable default value for empty object is `null`

Reply:
Yeah, my bad. Fixed!